builders/interact/substrate-api/.pages

```diff
@@ -4,5 +4,6 @@ nav:
   - index.md
```

Comment:
@eshaben should we do this alphabetically?

Reply:
It's ordered alphabetically now